frontend/libs/studio-components/src/components/StudioToggleableTextfield/StudioTextfieldToggleView/StudioTextfieldToggleView.tsx

```diff
@@ -4,21 +4,27 @@ import { StudioButton, type StudioButtonProps } from '@studio/components';
 import classes from './StudioTextfieldToggleView.module.css';
 import cn from 'classnames';
 
-export type StudioTextfieldToggleViewProps = StudioButtonProps;
+export type StudioTextfieldToggleViewProps = StudioButtonProps & {
+  customComponentIdInViewMode?: boolean;
+};
 
 export const StudioTextfieldToggleView = ({
   onClick,
   children,
   className: givenClass,
+  customComponentIdInViewMode = true,
```

Comment:
`customComponentIdInViewMode` is not suitable name here. This component is designed to be generic and can be used in various contexts, not just for componentId. The prop name is too closely tied to the specifics of the component id view. I recommend removing this class and instead passing a className as a prop from where we want to apply these styles.

Reply:
also removed and used className props